### PR TITLE
Added MekaLoot

### PR DESCRIPTION
**Pull request by paicasso**
## MekaLoot

Etherscan link: https://etherscan.io/address/0x1c653f203369ac9e67ba01fcf3ab5a509b6349d1

Website: https://opensea.io/collection/mekaloot

Twitter: https://twitter.com/meka_loot

### CODE REVIEW

File: utils/lists.ts [hunk 1]
```diff
@@ -611,4 +611,9 @@ export const derivativesList: Record<string, string>[] = [
     description: "The first anti-metaverse built on Ethereum",
     url: "https://upsidedownloot.com/",
   },
+  {
+    name: "MekaLoot",
```
Comment (thsnyzkn) [inline utils/lists.ts:615]: Please, place your added resource into correct alphabetical order. Thanks.

**Review by thsnyzkn — CHANGES_REQUESTED** [review]
Thanks for the contribution.

**Review by paicasso — COMMENTED** [review]
Done it

File: utils/lists.ts (hunk 1 above)
Comment (paicasso) [reply utils/lists.ts]: Done it. Does it have conflict with current version?

**Pull request merged**